src/security/forward.h

```diff
@@ -77,9 +81,17 @@ typedef struct gnutls_x509_crt_int Certificate;
 typedef struct notls_x509 Certificate;
 #endif
 
+// Macro to be used to define the C++ equivalent function of an extern "C"
+// function. The C++ function suffixed with the _cpp extension
+#define CtoCpp1(function, argument)                     \
```

Comment:
Keeping this definition in src/security/ does not address a "move CtoCpp1() definition" TODO in src/store/SwapMetaOut.h [comment](https://github.com/measurement-factory/squid/blob/40d6518de7ee641983d5d92bb6d44f08a077c6e2/src/store/SwapMetaOut.h#L17C1-L17C1). If this move is no longer necessary for this PR, please undo it. Otherwise, please consider moving CtoCpp1() definition to a more appropriate place, where SwapMetaOut.h and other future callers can reach it without referring to src/security (that this macro is unrelated to).

Reply:
Reverted

Reply:
> Reverted

Not yet.

Reply:
Huh? it is under USE_OPENSSL || USE_GNUTLS . Is that not what you meant by "revert"?

Reply:
> it is under USE_OPENSSL || USE_GNUTLS . Is that not what you meant by "revert"?

No, the current state does not address my change request. Please do see the original [change request](https://github.com/squid-cache/squid/pull/1463#discussion_r1334649322) for details that cover both the "revert" option (although I did not use that word) and its alternative.

N.B. If a change of the official code is truly reverted/undone, there will be no corresponding lines in the PR diff (because nothing has been modified compared to the official sources).